helm/csi-isilon/templates/node.yaml

```diff
@@ -161,6 +167,37 @@ spec:
               mountPath: /registration
             - name: driver-path
               mountPath: /csi
+        {{- if hasKey .Values "authorization" }}
+        {{- if eq .Values.authorization.enabled true }}
+        - name: karavi-authorization-proxy
+          imagePullPolicy: Always
```

Comment:
Set imagePullPolicy to {{ .Values.imagePullPolicy }}

Reply:
Fixed.